### ISSUE TITLE
Screen Reader: Profile Subsections: Invisible element receives focus after OTP field

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77493]
## Action Performed:
Precondition: User is logged in

1. Using TalkBack on Android, open the app
2. Swipe right to focus the Account tab and double-tap to activate it
3. Swipe right to focus the Profile button and double-tap to activate it
4. Swipe right to focus the Contact methods menu item and double-tap to activate it
5. Swipe right to focus an email address that requires verification and double-tap to activate it
6. Swipe right past the OTP cells and note what element receives focus

## Expected Result:
Focus should move in a meaningful sequence and only land on meaningful elements.
After the OTP cells container, focus should land on the first cell.

## Actual Result:
After the OTP cells, focus lands on an invisible element, announced as "Space".

## Area issue was found in:
Profile Subsections

## Failed WCAG checkpoints
WCAG 1.3.2 Meaningful Sequence
https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence

## User impact:
A meaningful sequence that matches a logical reading order helps people who rely on assistive technologies that read content aloud. The meaning evident in the sequencing of the information in the default presentation will be the same when the content is presented in spoken form.

## Suggested resolution:
Remove accessible={true} or focusable={true} from elements that are not visually interactive to ensure focus moves from the OTP cells container to the first cell.
https://appt.org/en/docs/react-native/samples/accessibility-focusability
https://reactnative.dev/docs/view#focusable-android
https://reactnative.dev/docs/accessibility#accessible

## Environment(s)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
<img width="1080" height="2195" alt="Image" src="https://github.com/user-attachments/assets/3db04a23-e7c1-44a2-8795-60e4314572df" />

https://github.com/user-attachments/assets/121cd981-9dbd-403c-af01-e329e882b1dc

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7008063
**Issue reported by:** NA
**Slack conversation:** NA
